python/lsst/pipe/tasks/photoCal.py

```diff
@@ -52,13 +52,11 @@ class PhotoCalConfig(pexConf.Config):
     )
     applyColorTerms = pexConf.Field(
         dtype=bool,
-        default=None,
-        doc=("Apply photometric color terms to reference stars? One of:\n"
-             "None: apply if colorterms and photoCatName are not None;\n"
-             "      fail if color term data is not available for the specified ref catalog and filter.\n"
-             "True: always apply colorterms; fail if color term data is not available for the\n"
-             "      specified reference catalog and filter.\n"
-             "False: do not apply."),
+        default=False,
+        doc=("Apply photometric color terms to reference stars?\n"
+             "True: always apply colorterms; fail if color term data is not available for the "
```

Comment:
Perhaps this would read more true if written:
```
True: attempt to apply color terms; fail if color term data...
```

Reply:
I've taken out the "always"; the "attempt" is implied by the failure description.

Reply:
Thanks - as mentioned in a reply above, I think your edit was undone by a subsequent edit. For the reasons elucidated above, I still prefer the use of `attempt to` here, but am happy to defer to your judgement on this.